gravitee-node-monitoring/src/test/java/io/gravitee/node/monitoring/DefaultProbeRegistryTest.java

```diff
@@ -0,0 +1,219 @@
+package io.gravitee.node.monitoring;
+
+import static io.gravitee.node.monitoring.DefaultProbeRegistry.SAFEGUARD_DELAY;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.*;
+
+import io.gravitee.node.api.healthcheck.Probe;
+import io.gravitee.node.api.healthcheck.ProbeManager;
+import io.gravitee.node.api.healthcheck.Result;
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.MockedStatic;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+/**
+ * @author Jeoffrey HAEYAERT (jeoffrey.haeyaert at graviteesource.com)
+ * @author GraviteeSource Team
+ */
+@ExtendWith(MockitoExtension.class)
+class DefaultProbeRegistryTest {
```

Comment:
you should add `@DisplayNameGeneration(DisplayNameGenerator.ReplaceUnderscores.class)` : new style